bin/Replication/Invoke-ReplicationSetup.ps1

```diff
@@ -0,0 +1,13 @@
+$password = ConvertTo-SecureString "dbatools.IO" -AsPlainText -Force
+$cred = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList "sqladmin", $password
+
+$PSDefaultParameterValues["*:SqlInstance"] = "localhost"
+$PSDefaultParameterValues["*:SqlCredential"] = $cred
+$PSDefaultParameterValues["*:Confirm"] = $false
+$PSDefaultParameterValues["*:SharedPath"] = "/shared"
+##$PSDefaultParameterValues["*:WarningAction"] = "SilentlyContinue"
+#$global:ProgressPreference = "SilentlyContinue"
+
+
+
+Invoke-DbaQuery -File .\bin\Replication\setup-test-replication.sql
```

Comment:
This file should be added to dataplat/appveyor-lab and referenced from that repository. You can include checking this repo out in the GH action.

Reply:
I actually ended up removing the need for this - so I'll remove the file from the repo! Sorry missed this one will remove it and the commented out references